kview/src/main/java/module-info.java

```diff
@@ -26,6 +26,7 @@
     requires dev.ikm.komet.classification;
     requires dev.ikm.komet.progress;
     requires org.carlfx.cognitive;
+    requires dev.ikm.tinkar.composer;
```

Comment:
Question: Is composer part of tinkar-core?

Reply:
it is its own dependency